Packs/ApiModules/Scripts/GSuiteApiModule/GSuiteApiModule.py

```diff
@@ -294,3 +295,32 @@ def strip_dict(args: Dict[str, str]) -> Dict[str, str]:
         :return: Dictionary with whitespaces and empty entries removed.
         """
         return {key: value.strip() for (key, value) in args.items() if value and value.strip()}
+
+
+''' HELPER FUNCTIONS '''
+
+
+def replace_token_text(text: str) -> str:
```

Comment:
replace with regex, get the prefix as input and move to CSP.

Reply:
Done